### ISSUE TITLE
Responsiveness

### ISSUE THREAD

**Issue by Syed0206** [Nandini80/Pill-Share#21]
@Nandini80 website is not responsive. Please assign me this issue under PHICSIT.

**Comment by Nandini80** [comment]
I have assigned this issue to you, but please work on one issue at a time.

**Comment by Nandini80** [comment]
@Syed0206 Are you working on this issue?

**Comment by Syed0206** [comment]
Yes. Please need some time till Friday I will complete and will submit PR

**Comment by Nandini80** [comment]
ok
